runtime/src/main/java/org/corfudb/runtime/collections/CorfuStreamEntry.java

```diff
@@ -121,36 +112,10 @@ private static OperationType getOperationType(@Nonnull SMREntry entry) {
                 operationType = OperationType.DELETE;
                 break;
             default:
-                throw new RuntimeException("SMRMethod "+entry.getSMRMethod()
+                throw new RuntimeException("SMRMethod " + entry.getSMRMethod()
                         + " cannot be translated to any known operation type");
         }
-        return operationType;
-    }
-
-    /**
-     *  Convert an SMREntry into a CorfuStreamEntry when the SMR arguments have clear types
-     *  (like if called from a commit callback)
-     * @param entry - the entry to convert from.
-     * @param epoch - the epoch this entry was written to/will be written to.
-     */
-    public static
-    CorfuStreamEntry fromSMREntry(@Nonnull SMREntry entry, final long epoch) {
-
-        long address = entry.getGlobalAddress();
-
-        OperationType operationType = getOperationType(entry);
-        Object[] args = entry.getSMRArguments();
-        if (args.length > 0) {
-            if (args.length > 1) {
-                CorfuRecord record = (CorfuRecord) args[1];
-                return new CorfuStreamEntry((Message)args[0],
-                        record.getPayload(), record.getMetadata(), 0, address, operationType);
-            }
-            // this would likely be the case with "remove" SMR method
```

Comment:
Consolidating the functions is ok, but would be good to retain these comments.

Reply:
Will do.